plugins/webp-uploads/hooks.php

```diff
@@ -679,7 +679,8 @@ function webp_uploads_img_tag_update_mime_type( string $original_image, string $
 			$image !== $original_image &&
 			'the_content' === $context &&
 			'image/jpeg' === $original_mime &&
-			'image/webp' === $target_mime
+			'image/webp' === $target_mime &&
+			! webp_uploads_picture_element_enabled() // Don't enqueue fallback JS if picture enabled.
 		) {
 			add_action( 'wp_footer', 'webp_uploads_wepb_fallback' );
```

Comment:
I just noticed that `webp_uploads_wepb_fallback` has a typo. It should be `webp_uploads_webp_fallback`! 

_Web Pee_ 🚽

Reply:
renamed in 47304d4709439eb503f73952ede90c15b50c18b4